tests/integration/test_video_input_worker.py

```diff
@@ -6,14 +6,21 @@
 import queue
 import time
 
+from modules.common.modules.camera import camera_factory
+from modules.common.modules.camera import camera_opencv
 from modules.video_input import video_input_worker
 from modules import image_and_time
 from utilities.workers import queue_proxy_wrapper
 from utilities.workers import worker_controller
 
 
+# Modify these settings as needed
 VIDEO_INPUT_WORKER_PERIOD = 1.0
-CAMERA = 0
+CAMERA = camera_factory.CameraOption.OPENCV
```

Comment:
Also failing here, but for some reason there is a conversion which results in this being `float` in the worker function.